.pre-commit-config.yaml

```diff
@@ -19,6 +21,7 @@ repos:
         args:
           - --remove
       - id: pretty-format-json
+        exclude: package(-lock)?\.json
```

Comment:
Would this affect `tsconfig.json` in any way?

Reply:
This isn't being synced with the frontend yet. There are some repo-specific hooks in the frontend so syncing them centrally is not feasible (yet).